ingestion/functions/parsing/peru/peru_test.py

```diff
@@ -20,108 +20,178 @@ def test_parse(self):
+                                               'geoResolution': 'Admin3',
+                                               'name': 'LIMA, LIMA, LIMA, Peru',
+                                               'geometry': {'latitude': -11.7855675478949,
+                                                            'longitude': -76.6271728665078}},
                                   'events': [{'name': 'confirmed',
                                               'value': 'Serological test',
                                               'dateRange': {'start': '03/30/2020Z', 'end': '03/30/2020Z'}}],
                                   'demographics': {'ageRange': {'start': 29.0, 'end': 29.0},
                                                    'gender': 'Female'}},
-                               {'caseReference': {'sourceId': 'place_holder_ID',
+                               {'caseReference': {'sourceId': 'placeholder_ID',
                                                   'sourceEntryId': 'bc45b71b005a96f32eeb97060616a999',
-                                                  'sourceUrl': 'place_holder_URL'},
-                                  'location': {'query': 'LIMA, LIMA, Lima Province, Peru'},
+                                                  'sourceUrl': 'placeholder_URL'},
+                                  'location': {'administrativeAreaLevel3': 'LIMA',
+                                               'administrativeAreaLevel2': 'LIMA',
+                                               'administrativeAreaLevel1': 'LIMA',
+                                               'geoResolution': 'Admin3',
+                                               'name': 'LIMA, LIMA, LIMA, Peru',
+                                               'geometry': {'latitude': -11.7855675478949,
+                                                            'longitude': -76.6271728665078}},
                                   'events': [{'name': 'confirmed',
                                               'value': 'Serological test',
                                               'dateRange': {'start': '03/30/2020Z', 'end': '03/30/2020Z'}}],
                                   'demographics': {'ageRange': {'start': 44.0, 'end': 44.0},
                                                    'gender': 'Female'}},
-                               {'caseReference': {'sourceId': 'place_holder_ID',
+                               {'caseReference': {'sourceId': 'placeholder_ID',
                                                   'sourceEntryId': '0e2a1928ddd07d99978758e86b034131',
-                                                  'sourceUrl': 'place_holder_URL'},
-                                  'location': {'query': 'LIMA REGION, Peru'},
+                                                  'sourceUrl': 'placeholder_URL'},
+                                  'location': {'administrativeAreaLevel1': 'LIMA',
+                                               'name': ' , Peru',
```

Comment:
Shouldn't name be Lima, Peru?

Reply:
All 3 administrative areas are Lima, so calling it ` Lima, Lima, Lima, Peru` differentiates it from ` Barranco, Lima, Lima, Peru` for example

Reply:
what I mean is, name above is ' , Peru' which seems weird